scripts/add_degree.py

```diff
@@ -0,0 +1,101 @@
+           'Master of IT ( MSIT, MScIT, MScIT, MScIT or MSc IT)',
+           'Master of Jurisprudence (MJ or MJur)',
+           'Master of Laws (LLM or LLM)',
+           'Master of Studies in Law (MSL)',
+           'Master of Landscape Architecture (MArch)',
+           'Master of Letters (MLitt)',
+           'Master of Liberal Arts (MA, ALM, MLA, MLS or MALS)',
+           'Master of Library and Information Science (MLIS)',
+           'Master of Management (MM)',
+           'Master of Mathematics (or MMath)',
+           'Master of Mathematical Finance',
+           'Master of Medical Science',
+           'Master of Music (MM or MMus)',
+           'Master of Occupational Therapy (OT)',
+           'Master of Pharmacy (MPharm or MPharm)',
+           'Master of Philosophy (MPhil)',
+           'Master of Physics (MPhys)',
+           'Master of Physician Assistant Studies',
+           'Master of Political Science',
+           'Master of Professional Studies (MPS or MPS)',
+           'Master of Public Administration (MPA)',
+           'Master of Public Affairs (MPAff)',
+           'Master of Public Health (MPH)',
+           'Master of Public Management',
+           'Master of Public Policy (MPP)',
+           'Master of Quantitative Finance',
+           'Master of Rabbinic Studies (MRb)',
+           'Master of Real Estate Development',
+           'Master of Religious Education',
+           'Master of Research - MSc(R)',
+           'Master of Sacred Theology (STM)',
+           'Master of Sacred Music (MSM)',
+           'Master of Science (MSc, MSc, MSci, MSi, ScM, MS, MSHS, MS, Mag, Mg, Mgr, SM, or SM)',
+           'Master of Science in Education',
+           'Master of Science in Engineering (MSE)',
+           'Master of Science in Finance (MFin)',
+           'Master of Science in Human Resource Development (HRD or MSHRD)',
+           'Master of Science in Information Systems Management (MSMIS)',
+           'Master of Science in Information Systems (MSIS)',
+           'Master of Science in Information Technology (MSIT, MScIT, MScIT, MScIT or MSc IT)',
+           'Master of Science in Nursing (MSN)',
+           'Master of Science in Project Management (MSPM)',
+           'Master of Science in Management (MSc or MSM)',
+           'Master of Science in Leadership (MSL)',
+           'Master of Science in Supply Chain Management (SCM or MSSCM)',
+           'Master of Science in Taxation',
+           'Master of Science in Teaching (MST)',
+           'Master of Social Work (MSW)',
+           'Master of Social Science (MSSc)',
+           'Master of Surgery (ChM or MS, as well as MCh and MChir)',
+           'Master of Studies (MSt or MSt)',
+           'Master of Theology (ThM or MTh)',
+           'Master of Theological Studies (MTS)',
+           'Master of Urban Planning',
+           'Master of Veterinary Science (MVSC or MVSc)' ]
```

Comment:
Not these degrees.. you could just copy whatever Linkedin has. It should be something like:
```
PhD
MS
M.Tech
12th
10th
```